osf/management/commands/check_crossref_dois.py

```diff
@@ -34,33 +42,37 @@ def check_crossref_dois(dry_run=True):
         is_published=True
     ).exclude(date_published__gt=timezone.now() - time_since_published)
 
-    if not preprints_with_pending_dois:
```

Comment:
You have an outdated comment above - 
    # Create one enormous url to check if all our pending crossref DOIs are good, then set all identifers

Can you update to explain why you're batching preprints like this?

Reply:
It's actually 2 reasons
1) The crossref response is paginated by 20 items, so why don't we page through? Because of 2.
2) Crossref limits the length of URIs so we are limited in the number of ids we can query at the same time, so one query per page is easiest. If we could query all and page through them it might be worth it, but we can't.

Reply:
yes, I understand why, just asking you to update with a code comment to reflect this!  You are no longer creating one giant url

Reply:
It's [not just crossref](https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers#417184), they're following standard practice.